### ISSUE TITLE
Exception: Illuminate\Contracts\Container\BindingResolutionException

### ISSUE THREAD

**Issue by 01Kuzma** [OFFLINE-GmbH/oc-speedy-plugin#12]
Hi,
I'm running Winter CMS locally on PHP 8.1 and receiving this error:
```

Target [Illuminate\Routing\RouteCollectionInterface] is not instantiable while building [OFFLINE\Speedy\Classes\Middleware\CDNMiddleware, Illuminate\Routing\UrlGenerator].

...\vendor\laravel\framework\src\Illuminate\Container\Container.php line 1103

    Type

    Undefined
    Exception

    Illuminate\Contracts\Container\BindingResolutionException

```

![image](https://user-images.githubusercontent.com/569504/222118435-b9a13ee8-1b39-43a6-89d4-9f98aa38ca7c.png)
Is it related to PHP 8.1 ?

Thank you!

**Comment by 01Kuzma** [comment]
Any thoughts?

**Comment by tobias-kuendig** [comment]
Can you try if `route()->to('/')` works for this line instead of using the UrlGenerator? Also remove it from the constructor.

https://github.com/OFFLINE-GmbH/oc-speedy-plugin/blob/develop/classes/middleware/CDNMiddleware.php#L24

**Comment by 01Kuzma** [comment]
HI,
I've changed it to `$this->baseUrl      = $this->url()->to('/');`
But the error is the same.

"Also remove it from the constructor."
I haven't found it, could you please point?

**Comment by 01Kuzma** [comment]
@tobias-kuendig , any thoughts?

**Comment by 01Kuzma** [comment]
That works!
Thank you!

**Comment by tobias-kuendig** [comment]
Thank you for the report, this has been released as v1.0.12

**Comment by 01Kuzma** [comment]
Thank you for the quick fix!
